content/docs/user-guide/components/reference/terrain/surface-material-list.md

```diff
@@ -15,13 +26,18 @@ Select the surface type tag using the **SurfaceTag** drop-down menu, then assign
 
 [Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)
 
+## Dependencies
+
+[Axis-Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape)
+
 ## Properties
 
 ![Terrain Surface Materials List component properties](/images/user-guide/components/reference/terrain/terrain-surface-materials-list-component.png)
 
 | Property | Description | Values | Default |
 |-|-|-|-|
-| **Gradient to Material Mappings** | An array of [surface tags](/docs/user-guide/gems/reference/environment/surface-data) and material assets to map together. |  |  |
+| **Default Material** | The default material to fall back to where no other material surface mappings exist.<br><br>**NOTE:** The default material doesn't blend with other materials because it has no surface weight. The primary intended uses of the default material are either as an error material to see every place that a detail material hasn't been mapped, or to quickly cover an entire terrain surface with a single material without setting up more complicated mappings. | Material Asset | None |
```

Comment:
```suggestion
| **Default Material** | The default material to fall back to when no other material surface mappings exist.<br><br>**NOTE:** The default material doesn't blend with other materials because it has no surface weight. The primary intended uses of the default material are either as an error material to see every place that a detail material hasn't been mapped, or to quickly cover an entire terrain surface with a single material without setting up more complicated mappings. | Material Asset | None |
```

Reply:
Done